### ISSUE TITLE
Donation Validation?

### ISSUE THREAD

**Issue by hellnest** [5andr0/PogoLocationFeeder#309]
Where can i validate my donation to got validated feed? sorry didn't know where the support forum is.

**Comment by ifbatmanwasreal** [comment]
Yeah I'm wondering this too. What's better about the validated feed? I'll donate for that. I'm catching like 10% of the pokemon I snipe.

**Comment by arnachy** [comment]
same problem

**Comment by hellnest** [comment]
I already donated but don' knowhow to confirm my donation. It's not a big money but if it worth the penny i will donate more.

**Comment by Sheetka** [comment]
wondering this too
